src/user/AddressBookController.ts

```diff
@@ -9,10 +9,12 @@ import BaseController, { BaseConfig, BaseState } from '../BaseController';
  *
  * @property address - Hex address of a recipient account
  * @property name - Nickname associated with this address
+ * @property importTime - Data time when an account as created/imported
  */
 export interface ContactEntry {
   address: string;
   name: string;
+  importTime?: number;
```

Comment:
Huh. Why is this type here? :thinking: I just went to see where it's used in this file, and.... it isn't.

This should be declared where it's used. We can fix that in a separate PR though.

Reply:
I will bump this to another PR

Reply:
Hi there. I need to know when my account was created. Was it implemented? Is this information is logged in the extension log files? Would really appreciate your help. Thanks.